src/main/java/com/codingnomads/betty/data/api/TwitterSourceMinerRepository.java

```diff
@@ -37,4 +37,37 @@ public TwitterSourceMinerRepository(TwitterConfig twitterConfigurer) {
 
         return tweets;
     }
+
+    public List<Status> searchTweetFromAccounts(String keyword, String startDate) {
+        Queue<String> queries = new LinkedList<>();
+        queries.add("Chelsea -Arsenal from:ChelseaFC");
```

Comment:
Is there anyway we can make these dynamic fields or pojos instead of hardcoding the value? If what you need is the latest matches, you can use the query I built in the JPA repository to extract the latest ones. Once you rebase of course

Reply:
Yep that is where the QueryConstructor class will come in and construct a list of queries using the list of twitter accounts/hashtags related to each team, whose information will come from the pojos. This is coming next on my todo list!